injectproxy/utils.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 The Prometheus Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package injectproxy
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+)
+
+func PrometheusAPIError(w http.ResponseWriter, errorMessage string, code int) {
```

Comment:
One last thing occurs to me, can we make this function _nit_ exported? It's not used out of the package, so there is no need for it to be exported.